include/Utils.hpp

```diff
@@ -126,6 +127,14 @@ const static constexpr char* interface =
     "xyz.openbmc_project.Association.Definitions";
 } // namespace association
 
+namespace pgood
+{
+const static constexpr char* busname = "org.openbmc.control.Power";
```

Comment:
`const static constexpr` ?

Never seen that before, surprised it compiled since isnt `constexpr` imply `const` already?